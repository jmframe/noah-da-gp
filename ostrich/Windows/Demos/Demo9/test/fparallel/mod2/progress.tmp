          10  6.222321868440298E-018  1.769458736134766E-003
