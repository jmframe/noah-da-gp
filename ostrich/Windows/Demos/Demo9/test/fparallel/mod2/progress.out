 solve
     1      0.232E+04      0.178E-02      0.000E+00
     2      0.360E-05      0.176E-02      0.000E+00
     3      0.135E-06      0.176E-02      0.000E+00
     4      0.394E-08      0.179E-02      0.000E+00
     5      0.135E-09      0.178E-02      0.000E+00
     6      0.462E-11      0.181E-02      0.000E+00
     7      0.159E-12      0.180E-02      0.000E+00
     8      0.545E-14      0.177E-02      0.000E+00
     9      0.188E-15      0.176E-02      0.000E+00
    10      0.622E-17      0.177E-02      0.000E+00
 done
 finished
